src/ApiBundle/Form/UserType.php

```diff
@@ -32,7 +32,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('email', EmailType::class)
             ->add('dob', DateType::class, array('widget' => 'single_text', 'format' => 'M/d/y'))
             ->add('username', TextType::class)
-            ->add('password', PasswordType::class, array('data' => ''))
+            ->add('password', TextType::class)
```

Comment:
changed it back to TextType? 
it was chnaged to PasswordType in last PR
https://github.com/nerdapplabs/authOauth/pull/9/files#diff-3683af68b1a14d46bf30b457a8ec4ddfR31